core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java

```diff
@@ -315,6 +315,7 @@ public void success( SerializedResultSet rs ) {
             hideWaitingDialog();
             displayErrorMessage( e );
           }
+          previewResultsDialog.center();
```

Comment:
Should this not be added in line 306?
In case of error, does it make sense to show the dialog?